src/test/java/BaseTest.java

```diff
@@ -24,13 +28,36 @@ static void setupClass() {
 
     @BeforeMethod
     public void setUpBrowser() {
-        driver = new ChromeDriver();
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--disable-notifications");
+        driver = new ChromeDriver(options);
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
         driver.get(url);
+        driver.manage().window().maximize();
     }
 
     @AfterMethod
     public void tearDown() {
         driver.quit();
     }
-}
+
+    public static void enterEmail(String email) {
+        WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
+        emailInput.click();
+        emailInput.sendKeys(email);
+    }
+
+    public static void enterPassword(String password) {
+        WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
+        passwordInput.click();
+        passwordInput.sendKeys(password);
+    }
+
+    public static void loginSubmit() {
+        WebElement submitLogin = driver.findElement(By.cssSelector("button[type='submit']"));
+        submitLogin.click();
+    }
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._

Reply:
ok, thanks, I've committed my changes. I don't know how to link to a commit in an existing branch, sorry)